src/notify.js

```diff
@@ -71,6 +83,11 @@ Notify.isSupported = function(perm) {
         throw new Error('You must only call this before calling Notification.requestPermission(), otherwise this feature detect would trigger an actual notification!');
     }
 
+    if (Notify.serviceWorkerSupported()) {
```

Comment:
I still lean towards treating them as the same thing really... From the end user perspective, I just want to see a notification when I get a new message, etc, I don't care which API is used or where it came from, be it Push, Web Socket, Web Worker, etc...